src/JointAxis.cc

```diff
@@ -151,6 +154,20 @@ Errors JointAxis::Load(ElementPtr _sdf)
     this->dataPtr->dissipation = limitElement->Get<double>("dissipation",
         this->dataPtr->dissipation).first;
   }
+
+  // Load mimic joint details if provided.
+  if (_sdf->HasElement("mimic"))
+  {
+    sdf::ElementPtr mimicElement = _sdf->GetElement("mimic");
```

Comment:
We should prefer using `FindElement` instead of the combination of `HasElement` and `GetElement`.

Reply:
Done https://github.com/gazebosim/sdformat/pull/1166/commits/8e4da216ac5f3c08b4ac9eb87863369cf9076466